packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -165,26 +180,27 @@
 		<nav
 			aria-label="On this page"
 			transition:slide={{ axis: 'y', easing: expoOut, duration: $reduced_motion ? 0 : 400 }}
+			on:introstart={() => mobile_menu_open && (mobile_z_index = 101)}
+			on:outrostart={async () => {
+				await tick();
+
+				if (!mobile_menu_open && $nav_open) {
+					mobile_z_index = 2;
```

Comment:
nit: I'd introduce a constant for `2` since we use it in a few places

Reply:
Done